### ISSUE TITLE
I love this engine. Question about MDR models

### ISSUE THREAD

**Issue by CuteNoiRe** [ec-/Quake3e#252]
How to use the MDF format for character models, is it a skeletal format?

**Comment by ensiform** [comment]
Models are not used for collision in this engine, you would need to write all of that yourself.

**Comment by KuehnhammerTobias** [comment]
Both are skeletal format. Personally I recommend IQM over MDR (higher FPS + lower size). IQM works best with GPU vertex skinning (needs Opengl2 renderer).
Anyways, the person of interest is zturtleman, he can also explain how to rework things in the cgame module to set up IQM models.
If you have Discard installed you can talk to him there. He didn't only implement the IQM format, he also made a really cool model creation tool called Maverick Model 3D (with IQM support).
I recommend to get in contact to him, because once I had the same question(s) like you, and he helped me really a lot.
Here is a link  to a very technical but interesting conversation I had with him:
https://discourse.ioquake.org/t/current-status-of-iqm-md5-models/1153
Where you can find zturtleman:
https://github.com/zturtleman
https://twitter.com/zturtleman
https://[clover.moe](https://clover.moe/)/

Basically he did most model imrovements MDR/MDS/IQM for his Spearmint engine, which is based on ioquake3. So all his work will most likely apply to Q3e as well.
I really hope you will success using those model formats. Good luck!
